src/Resources/app/storefront/src/js/filter-category-select.plugin.js

```diff
@@ -184,28 +314,32 @@ export default class FilterCategorySelectPlugin extends FilterBasePlugin {
         return DomAccess.querySelectorAll(this.el, `${this.options.checkboxSelector}:checked`, false);
     }
 
+    /**
+     *
+     * @param {array} filter
```

Comment:
If you know the structure of the filter, please add it to the docblock, otherwise this can be ignored.

Reply:
I am not sure about its structure, I tried but got no clue for now.